### PR TITLE
Update Repo Nomenclature to "Collab Space"

### LINKED ISSUE #253: Task list for converting from WG to Collab Space
The CPC would like to merge Collab Spaces and Working Groups (see rationale in [#1110](https://github.com/openjs-foundation/cross-project-council/issues/1110)).

iirc we had discussed that and agreed to it in a WG meeting a while back, but I couldn't find a note of it in the meeting minutes. So this is a formal request for the WG to OK this migration.
```[tasklist]
### Tasks
- [ ] Update all references from WG to Collab Space
- [ ] Document Decisions that have been delegated to the Collab Space
- [ ] Add to list of OpenJS Collab Spaces
```

### PR DESCRIPTION
**Pull request by jorydotcom**
replaces Working Group nomenclature with Collab Space

Closes #253

### CODE REVIEW

**Review by rmarkins-godaddy — APPROVED** [review]

**Review by rxmarbles — APPROVED** [review]

**Review by UlisesGascon — APPROVED** [review]

File: MEMBER_REPRESENTATION.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ The following document lists OpenJS Foundation's memberships and liaison relatio
   * Tobie Langel
 * [Open Policy Alliance]
   * Tobie Langel
-* [Unicode MessageFormat Working Group]
```
Comment (ctcpip) [inline MEMBER_REPRESENTATION.md:25]: this is a mistake.  I'll fix with other updates

**Review by ctcpip — APPROVED** [review]
s/b `collaboration space`

File: MEMBER_REPRESENTATION.md (hunk 1 above)
Comment (ctcpip) [reply MEMBER_REPRESENTATION.md]: fixed

File: GOVERNANCE.md [hunk 2]
```diff
@@ -1,38 +1,38 @@
...
 added on an ongoing basis.
 
 * Invited to all meetings
 * Can participate in [consensus seeking process](#consensus-seeking-process)
-* Counted towards quorum in [Working Group Meetings](#working-group-meetings)
+* Counted towards quorum in [Collaboration Space Meetings](#collaboration-space-meetings)
 * Participates in voting
 
-## Working Group Membership
+## Collaboration Space Membership
```
Comment (ljharb) [inline GOVERNANCE.md:17]: this is going to break links; we should probably add an `<a id="working-group-membership"></a>` and similar throughout to preserve them

Comment (ctcpip) [reply GOVERNANCE.md:17]: well, technically going to break fragments, which is not nearly as bad.  I'd rather let them break than add html to md  🤷

Comment (ljharb) [reply GOVERNANCE.md:17]: fragments are still part of a URL :-/

Comment (ctcpip) [reply GOVERNANCE.md:17]: I just checked and there are no links to this in the openjs org's repos.  are there links elsewhere that you know of?

Comment (ljharb) [reply GOVERNANCE.md:17]: nope - but the potential exists. just like semver is about theoretical breakage not actual breakage, URL preservation is about potential breakage not just known breakage.

Comment (ctcpip) [reply GOVERNANCE.md:17]: I find this semi-breakage an acceptable sacrifice

**Review by ljharb — APPROVED** [review]

**Pull request merged**